viewer/packages/cad-model/src/wrappers/CogniteCadModel.ts

```diff
@@ -51,6 +51,25 @@ export class CogniteCadModel implements CdfModelNodeCollectionDataProvider {
     return getDistanceToMeterConversionFactor(this.modelUnit);
   }
 
+  /**
+   * Sets the model visibility.
+   * @default true
```

Comment:
It does not look like this function has a default value.

Reply:
When a model is added into viewer, the visibility by default is true